src/modules/currency/__tests__/formatNumber.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { test } from '@playwright/test';
+import formatNumber from '../formatNumber';
+import { assertScriptText, injectScript } from '../../../blackbox/utils';
```

Comment:
change: can we please use an absolute path here? what is the black box here, can we rename it in better way

Reply:
Blackbox is a directory for playwright utils. 
don't have the support for alias paths right now in the setup. There were some issues on enabling it from tsconfig and eslint so have kept it this way. 
As this is common across the entire project, will be pushing a separate PR to address this.

Reply:
`Blackbox is a directory for playwright utils.` can we rename this for better clarity, why not just mention` playwrite-test-utils`  or something like this.

I don't think, blackbox is a common ward for this kind of scenarios

Reply:
'blackbox' here means that the developer need not know the internal implementations on how the test cases are being rendered in UI but can focus on the input and output to the system. 
Keeping it to a generic term here also gives us the advantage of using any other framework like `puppeteer` in future if we have to. Also, this term has been inspired from existing products in razorpay. @avinashdvv

Reply:
I am still not aligned with the `blackbox`, you can just write this as `test-utils.ts`, something like this. Since checkout and other repos also following this approch, i am ok with this.

But in future, if you are writing the wiki like this https://github.com/razorpay/frontend-sme-dashboard/blob/master/docs/folder-structure.md

Please mention,  what does `blackbox` means

Reply:
sure, will add this in the documentation. 
Not aligned with `test-utils` here as it can be confused with unit test cases utils too.